typescript/libs/node_client.py

```diff
@@ -111,7 +111,7 @@ def makeTimeoutMsg(self, cmd, seq):
         }
         return timeoutMsg
 
-    def sendCmd(self, cb, cmd, seq):
+    def sendCmd(self, cmd, seq, cb):
```

Comment:
You have derived a class from CommClient and overridden this sendCmd method, you have have changed the meaning of the first 3 params (which are "self, cmd, cb" in the base class).  You should keep the matching parameters the same in the derived class (as they were in the original code).